actions/setup/action.yml

```diff
@@ -17,6 +12,13 @@ inputs:
     required: false
     default: '1.22'
 
+  cacheKey:
+    description: Custom cache key used
+    required: false
+    # NOTE: some projects cannot use **/go.sum because: https://github.com/actions/runner/issues/449
+    # In such cases, try providing the cache key and set it to e.g. ${{ inputs.go-version }}-${{ hashFiles('go.sum', '*/go.sum') }}
+    default: -${{ hashFiles('**/go.sum') }}
```

Comment:
The value this is appended to already contains a dash so it would cause a double dash :)

Reply:
Whoops 😄 fixed!